<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1355</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SIFT Demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="browse_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>400</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Browse template</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toggle_cam_button">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>400</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Enable camera</string>
    </property>
   </widget>
   <widget class="QLabel" name="template_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="live_image_label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>100</y>
      <width>640</width>
      <height>240</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>410</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Homography</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1355</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
